render-test/runner.cpp

```diff
@@ -783,6 +832,8 @@ TestRunner::Impl::Impl(const TestMetadata& metadata)
 bool TestRunner::run(TestMetadata& metadata) {
     AllocationIndex::setActive(false);
     AllocationIndex::reset();
+    ProxyFileSource::setTrackingActive(false);
```

Comment:
Nit: To be consistent with `AllocationIndex` can we rename this to `setActive` and `reset`?